_content/booklets/sixty-songs-of-milarepa_milarepa.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Sixty Songs of Milarepa" 
+authors:
+  - milarepa
+editor: "Bhikkhu Khantipālo"
+translator: "Garma C.C. Chang"
+external_url: "https://www.bps.lk/olib/wh/wh095_Chang_Sixty-Songs-of-Milarepa.pdf"
```

Comment:
Always the html version for wheels: https://www.bps.lk/olib/wh/wh095_Chang_Sixty-Songs-of-Milarepa.html